topology/sli/topology-interface.sli

```diff
@@ -158,6 +158,11 @@
     
   /get [/masktype /literaltype] {exch cvdict_M exch get} def
   /get [/masktype /arraytype] {exch cvdict_M exch get} def
+  
+  /SelectNodesByMask [/doubletype /doubletype /masktype /integertype]
+    /SelectNodesByMask_L_M_d_d load
```

Comment:
The anchor position should be supplied as an array or doublevector, so that we can support 2D and 3D with one function.